07-map/for3.go

```diff
@@ -0,0 +1,17 @@
+package main
+
+import "fmt"
+
+func main() {
+	ages := map[string]int{
+		"Baby":   0,
+		"Justin": 45,
+		"Donald": 70,
+		"Canada": 150,
+	}
+	// START OMIT
+	for key, _ := range ages {
```

Comment:
there's no need to put the blank identifier here. it's just for the `_, value` case

because as you've shown, you can just do `for key := range ages`